### PR TITLE
Post tsuru @ LoPUG

### PR DESCRIPTION
**Pull request by saliceti**

### CODE REVIEW

File: _posts/2015-03-02-tsuru-at-london-paas-user-group.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+date: '2015-02-28 20:50:00'
+date_gmt: '2015-02-28 20:50:00'
+categories:
+- Talk
+tags:
+- talk
+comments: []
+---
+
+On Feb 26th tsuru was invited at London PaaS User group.
```
Comment (fsouza) [inline _posts/2015-03-02-tsuru-at-london-paas-user-group.md]: Should it be "was invited for a talk at ..."?

**Comment by fsouza** [comment]
Hey @saliceti, thanks for contributing and for speaking about tsuru. I think we could add some sort of summary of the talk, and connect this summary to the linked slide deck. What do you think?

**Comment by saliceti** [comment]
"tsuru was invited" is an expression, it makes it look like tsuru is a person!

**Comment by fsouza** [comment]
Ah, TIL! Thanks for this. I think it's nice to merge now. Just want to make sure others are ok with that.

**Comment by tarsisazevedo** [comment]
I think we can merge it :) Thanks @saliceti

**Comment by fsouza** [comment]
And it's live! http://blog.tsuru.io/guest-post/2015/03/05/tsuru-at-london-paas-user-group.html

**Pull request merged**